core/profiles/standard/standard.install

```diff
@@ -224,7 +229,27 @@ function standard_install() {
       'body' => array(
         LANGUAGE_NONE => array(
           array(
-            'value' => st('This is your first post! You may edit or delete it.'),
+            'value' => st('This is your first post! You may <a href="node/2/edit">edit or delete it</a>.'),
```

Comment:
Same as previous comment, I don't think that we can be 100% sure that the ID of this node will always be `2`?

Reply:
See my previous reply.